web/src/components/Upload.vue

```diff
@@ -6,22 +6,6 @@ import FileList from '@girder/components/src/components/Presentation/FileUploadL
 import { UPLOAD_CSV, UPLOAD_EXCEL } from '@/store/actions.type';
 import { REMOVE_DATASET } from '@/store/mutations.type';
 
-const sampleTypes = [
-  { name: 'Serum', value: 'serum' },
-  { name: 'Urine', value: 'urine' },
-  { name: 'Tissue Extract', value: 'tissueextract' },
-  { name: 'Media', value: 'media' },
-  { name: 'Other', value: 'other' },
-];
-
-const dataTypes = [
-  { name: 'NMR Concentrations', value: 'nmr' },
-  { name: 'conc', value: 'conc' },
-  { name: 'GC/MS', value: 'gcms' },
-  { name: 'LC/MS', value: 'lcms' },
-  { name: 'Other', value: 'other' },
-];
-
```

Comment:
Is this a temporary removal until we can add backend support?  Or did Tom change his mind about wanting this?  Just curious.

Reply:
This is unclear. As far as I am aware it has no analytic value, so this type of information can always be placed in the name/description. I think it's good to remove for now and put it back if/when we know it's needed.

Reply:
in the end we always have the git history